src/Common/Core/Impl/OS/ProcessServices.cs

```diff
@@ -5,24 +5,12 @@
 
 namespace Microsoft.Common.Core.OS {
     public sealed class ProcessServices : IProcessServices {
-        private static IProcessServices _instance;
-
-        public static IProcessServices Current {
-            get {
-                if (_instance == null) {
-                    _instance = new ProcessServices();
-                }
-                return _instance;
-            }
-            internal set { _instance = value; }
-        }
-
-        public void Start(ProcessStartInfo psi) {
-            Process.Start(psi);
+        public Process Start(ProcessStartInfo psi) {
+            return Process.Start(psi);
         }
 
-        public void Start(string path) {
-            Process.Start(path);
+        public Process Start(string path) {
```

Comment:
It is huge and derives from Component... which derives from MarshalByRefObject and so on. I think no